src/System.Runtime/tests/System/StringGetHashCodeTests.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using Xunit;
+
+namespace System.Tests
+{
+    public class StringGetHashCodeTests : RemoteExecutorTestBase
+    {
+        [Fact]
```

Comment:
Comment like
```c#
// Ensure that string hash codes are randomized by getting the hash code for the same string in two processes
// and confirming it is different (modulo possible values of int).
// If the legacy hash codes are being returned, it will not be different.
```